src/PrivateDns/PrivateDns/help/Set-AzPrivateDnsZone.md

```diff
@@ -0,0 +1,228 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.PrivateDns.dll-Help.xml
+Module Name: Az.PrivateDns
+online version: https://docs.microsoft.com/en-us/powershell/module/az.privatedns/set-azprivatednszone
+schema: 2.0.0
+---
+
+# Set-AzPrivateDnsZone
+
+## SYNOPSIS
+Updates a Private DNS zone from a resource group.
+
+## SYNTAX
+
+### Fields (Default)
+```
+Set-AzPrivateDnsZone -ResourceGroupName <String> -Name <String> [-Tags <Hashtable>]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceId
+```
+Set-AzPrivateDnsZone [-Tags <Hashtable>] -ResourceId <String> [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### Object
+```
+Set-AzPrivateDnsZone -PrivateZone <PSPrivateDnsZone> [-Overwrite] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Remove-AzPrivateDnsZone** cmdlet permanently deletes a private Domain Name System (DNS) zone from a specified resource group.
```

Comment:
Still the description for remove.  Please read over all of these files carefully as our customers use them to understand our cmdlets. Also please look at them in Github to ensure the formatting is what you expect.

Reply:
thanks for pointing this out. Have updated the entire doc for set cmdlet.